ios/Components/DropInModule.swift

```diff
@@ -47,7 +47,7 @@ internal final class DropInModule: BaseModule {
             }
 
             do {
-                let address: LookupAddressModel = try address.toJson()
+                let address: LookupAddressModel = try address.decode()
```

Comment:
Nitpic: Variable shadowing
```suggestion
                let addressModel: LookupAddressModel = try address.decode()
```

Reply:
[4513c5b](https://github.com/Adyen/adyen-react-native/pull/542/commits/4513c5b173e7e9d053e941bfc33777b423d7404a)